api/NodeList.json

```diff
@@ -163,7 +163,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "8"
```

Comment:
`NodeList` is what's used for `childNodes`, but the `api.Node.childNodes` entry says IE5 and I've confirmed it's in IE7 at least. So what kind of object did IE return for `childNodes` before? I suspect this is just a problem of interface object not being exposed, and this should be set to an earlier version.

Reply:
I've added a custom test for this, and have updated the results manually!